README.adoc

```diff
@@ -275,24 +275,24 @@ Once the configuration file is created, configure your IDE to use it:
 
 === Visual Studio Code
 The Visual Studio Code extension provides custom formatter support for Microsoft Visual Studio Code.
-The extension using the https://code.visualstudio.com/api/references/vscode-api#DocumentFormattingEditProvider[`DocumentFormattingEditProvider`] API.
+The extension uses the https://code.visualstudio.com/api/references/vscode-api#DocumentFormattingEditProvider[`DocumentFormattingEditProvider`] API.
 Once installed it may be activated by using the "`Format Document`" action available in the editor context menu or from the Command Palette.
 
-To install the extension select "`Install from VSIX`" in the extensions panel and choose the `spring-javaformat-vscode-extension` vsix file.
+To install the extension, select "`Install from VSIX`" in the extensions panel and choose the `spring-javaformat-vscode-extension` vsix file.
 You can download the latest version from https://repo1.maven.org/maven2/io/spring/javaformat/spring-javaformat-vscode-extension/{release-version}[Maven Central].
 
 
 
 === About the Conventions
-Most of the coding conventions and style comes from the Spring Framework and Spring Boot projects.
-Spring Framework manually formats code, where as Spring Boot uses automatic formatting.
+Most of the coding conventions and style come from the Spring Framework and Spring Boot projects.
+Spring Framework manually formats code, whereas Spring Boot uses automatic formatting.
 
 
 
 === Indenting With Spaces
-By default tabs are used for indenting formatted code.
+By default, tabs are used for indenting formatted code.
 We strongly recommend that this default is not changed, especially for official Spring projects.
-If, however, you feel that you can't live with tabs then switching to spaces is the one configuration option that we do support.
+If, however, you feel that you can't live with tabs, then switching to spaces is the one configuration option that we do support.
```

Comment:
A comma isn't needed with "then" so this should either be "… live with tabs then switching …" or "… live with tabs, switching". Given the style elsewhere, I think the latter's better. We can make that tweak as part of merging.